src/network/UDPConnection.hpp

```diff
@@ -0,0 +1,39 @@
+/*
+** EPITECH PROJECT, 2024
+** RType
+** File description:
+** UDPConnection
+*/
+
+#ifndef UDP_CONNECTION_HPP_
+    #define UDP_CONNECTION_HPP_
+    #include <iostream>
+    #include <string>
+    #include <optional>
+    #include "boost/asio.hpp"
+
+using boost::asio::ip::udp;
+
+
+class UDPConnection {
```

Comment:
Could you document this class using doxygen comments ?

Reply:
In progress

Reply:
Done